src/cloud/cdn/fastly-vcl-whitelist.md

```diff
@@ -1,30 +1,38 @@
 ---
 group: cloud-guide
-title: Secure access to Magento Admin UI by client IP address
+title: Custom VCL for allowing requests
 redirect_from:
    - /cloud/configure/fastly-vcl-whitelist.html
 functional_areas:
   - Cloud
   - Setup
 ---
 
-The following example shows how to use a custom VCL snippet with a [Fastly Access Control List (ACL)](https://docs.fastly.com/guides/access-control-lists/about-acls) to secure access to the Magento Admin UI for a {{ site.data.var.ece }} project environment by client IP address. When you add the custom VCL snippet, Fastly allows only requests from IP addresses included in the ACL.
+
+You can use the a Fastly Edge ACL list in combination with custom VCL code snippet to filter incoming requests and allow access by IP address. The ACL list specifies the IP addresses to allow.
+
+Create an allowlist to limit access to your Staging environment so that only requests from specified IP addresses for internal developers and approved external services are permitted. You can also create an allowlist to secure access to the Magento Admin UI on Staging and Production environments.
```

Comment:
The "allowlist" looks like a reserved word, UI element, or something, unless it needs a space. Depending, this may need an emphasis of sorts. (I know what it is, but it is first occurrence on the page, so might need a little more attention.)